app.js

```diff
@@ -339,13 +339,40 @@ Promise.all([compilerFinder.find(), aws.initConfig(awsProps)])
             webServer.use(express.static(defArgs.staticDir, {maxAge: staticMaxAgeSecs * 1000}));
         }
 
+        // Removes last 3 octets for IPv6 and last octet for IPv4
+        // There's probably a better way to do this
+        morgan.token('gdpr_ip', req => {
```

Comment:
This should probably be some where else. util or something

Reply:
Great idea. I'll add some tests to it as I move it to utils :+1: